demo/src/main/java/com/example/demo/models/Video.java

```diff
@@ -0,0 +1,105 @@
+package com.example.demo.models;
+
+import jakarta.persistence.*;
+
+@Entity
+@Table(name = "videos")
+public class Video {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    public Long getId() {
+        return id;
+    }
+
+    @Lob
+    @Column(name = "data")
+    private byte[] data;
+
+    public void setData(byte[] data) {
+        this.data = data;
+    }
+
+    public byte[] getData() {
+        return data;
+    }
+
+    @Column(name = "rating")
+    private Integer rating;
+
+    public void updateRating() {
+        this.rating = Math.max(this.countLike - this.countDislike, 0);
+    }
+
+    public int getRating() {
+        return rating;
+    }
+
+    @Column(name = "identifier")
+    private String identifier;
+
+    public String getIdentifier() {
+        return identifier;
+    }
+
+    public void setIdentifier(String identifier) {
+        this.identifier = identifier;
+    }
+    @Column(name = "name")
+    private String name;
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    @Column(name = "countLike")
+    private Integer countLike;
+    public void setCountLike() {
+        this.countLike = 0;
```

Comment:
Better to not add any aggregators here.
The right way to get number of likes is to SELECT COUNT(*) FROM my_likes_table WHERE video_id=123...